src/main/java/com/nilhcem/fakesmtp/gui/listeners/AboutActionListener.java

```diff
@@ -56,7 +56,7 @@ public void actionPerformed(ActionEvent e) {
 		JEditorPane ep = new JEditorPane("text/html",
 			String.format("<html><body style=\"%s\">%s<br /><a href=\"%s\">%s</a></body></html>",
 				style, i18n.get("menubar.about.dialog"), link, link));
-
+		
```

Comment:
You probably saw already but: Done

On Sat, Jun 25, 2016 at 3:24 PM Vest <notifications@github.com> wrote:

> In
> src/main/java/com/nilhcem/fakesmtp/gui/listeners/AboutActionListener.java
> <https://github.com/Nilhcem/FakeSMTP/pull/47#discussion_r68492740>:
>
> > @@ -56,7 +56,7 @@ public void actionPerformed(ActionEvent e) {
> >  		JEditorPane ep = new JEditorPane("text/html",
> >  			String.format("<html><body style=\"%s\">%s<br /><a href=\"%s\">%s</a></body></html>",
> >  				style, i18n.get("menubar.about.dialog"), link, link));
> > -
> > +		
>
> Redundant change. Please remove spaces.
> @Nilhcem <https://github.com/Nilhcem> usually fixes them :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Nilhcem/FakeSMTP/pull/47/files/df778ca5e2f9e0a63415f6173ecc2ea23ce31bf7..0d5ef4719fce3a62bc5e744a4a4ed033f1268324#r68492740>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFr1-azNNsPFpauUtkJ9Vxz1XBylmySKks5qPYBWgaJpZM4I9S6K>
> .
>